frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -203,9 +203,6 @@ function testMode(actionType, editMode) {
       isReplyAllChecked = false;
       isForwardChecked = true;
     }
-
-    expect(wrapper.find("#to-field").props().selectedValues).toEqual(valEmailTo);
```

Comment:
I was going to say this is dropping a test to see that the to and cc fields are populated; but I think this is adequately tested elsewhere

Reply:
The input type is different so we can't actually access these values anymore. It seemed like it was tested elsewhere enough though.